VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_: AN400ECU Third_Party_Device

BO_ 1 MSGID_0X0CFFF048: 8 AN400ECU
 SG_ RPM : 0|16@0+ (1.0,0) [0|30000] "rpm"  Third_Party_Device
 SG_ TPS : 16|16@0+ (0.1,0) [0|100] "%"  Third_Party_Device

BO_ 2 MSGID_0X0CFFF048: 8 AN400ECU
 SG_ MAP : 0|16@0+ (0.01,0) [0|300] "psi/kPa"  Third_Party_Device
 SG_ Lambda : 16|16@0+ (0.01,0) [0|10] "lambda"  Third_Party_Device

BO_ 3 MSGID_0X0CFFF548: 8 AN400ECU
 SG_ Battery_Voltage : 0|16@0- (0.01,0) [0|22] "Volts"  Third_Party_Device
 SG_ Air_Temp : 16|16@0- (0.1,0) [-1000|1000] "C/F"  Third_Party_Device
 SG_ Coolant_Temp : 32|16@0- (0.1,0) [-1000|1000] "C/F"  Third_Party_Device
